tema27112019/src/main/java/ex4_Produs/Fructe.java

```diff
@@ -0,0 +1,12 @@
+package ex4_Produs;
+
+import java.util.Date;
+
+public class Fructe extends Perisable {
+  private String taraOrigine;
+
+    public Fructe(double price, String nume, String productor, UnitateDeMasura unitateDeMasura, String taraOrigine) {
+        super(price, nume, productor, unitateDeMasura);
```

Comment:
Fructele in general sult la kg. (Nu vreau ca cineva sa aiba posibilitatea de a crea fructe cu unitatea de masura bucata).
Sterge parametrul  UnitateDeMasura unitateDeMasura, din constructor ,  si apeleaza superconstructorul :
 **super(price, nume, productor, UnitateDeMasura.Kilogram,taraDeOrigine);**